antidox/README.md

```diff
@@ -4,8 +4,9 @@ In this directory user can manipulate the perspective and DLP API to detect requ
 Both of these tools are being used in an attempt to detect and prevent doxxing in online conversations,
 will be tested in wikipedia chat rooms as a staring point.
 
-1. Install library dependencies:
+1. Install [Bazel](https://docs.bazel.build/versions/master/install.html)
    ```shell
+   bazel test ...
     pip install -r requirements.txt
```

Comment:
remove the `pip install -r requirements.txt` bit?

Reply:
Done